app/controllers/TaxFreeChildcarePaymentsController.scala

```diff
@@ -27,21 +27,24 @@ import javax.inject.{Inject, Singleton}
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton()
-class TaxFreeChildcarePaymentsController @Inject() (cc: ControllerComponents, identify: AuthAction, eisConnector: NsNiConnector)(implicit ec: ExecutionContext)
-    extends BackendController(cc) {
+class TaxFreeChildcarePaymentsController @Inject() (
+    cc: ControllerComponents,
+    identify: AuthAction,
+    nsAndIConnector: NsAndIConnector
```

Comment:
can we use Nsi, nsi
its just nsAndI, NsAndI 
sounds a bit strange.
I ve seen it used on other repos on HMRC referred to as Nsi
it may be good to stick to that name